### ISSUE TITLE
gcc missing in docker build

### ISSUE THREAD

**Issue by janesp** [hyperledger/aries-endorser-service#30]
I found the following error while doing the edX course:

Executing this:

```
git clone https://github.com/bcgov/aries-endorser-service.git
cd aries-endorser-service/docker
./manage build
./manage start --logs
```

Failed in build process:

`error: command 'gcc' failed: No such file or directory`

**Comment by swcurran** [comment]
I found two different errors, but not that one.  I’m running on Linux, and the build was fine except:

- hard-coded `docker-compose` which only works on docker 1.x (vs. `docker compose` on v2.x)
- There are two “ports” entries in the “Author” section of the docker-compose.yml section.

Once I fixed those, everything seemed to build fine.

Not sure how `gcc` could be missing on your system but not mine, since the gcc should be running in Docker, not on the host machine.  It might be helpful to see the log showing on what step you saw the error.

**Comment by swcurran** [comment]
I’ll do a PR for the two issues I did see.

**Comment by swcurran** [comment]
Fixed the two errors in #31 .  Did not see an issue with `gcc`, but perhaps try again with the updates to the docker-compose.yml?  If you can create it again, please add details -- what platform are you on, and please include the full log of the run.

Thanks
